### PR TITLE
Added `MenuSeparator` component replacing `MenuDivider`

### PR DESCRIPTION
**Pull request by hirotomoyamada**
Related #3292

## Description

Added `MenuSeparator` component replacing `MenuDivider`.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/theme/src/components/menu.ts [hunk 1]
```diff
@@ -56,9 +56,15 @@ export const Menu: ComponentMultiStyle<"Menu"> = {
         bg: ["blackAlpha.50", "whiteAlpha.50"],
       },
     },
+
```
Comment (Itsuki54) [inline packages/theme/src/components/menu.ts:59]: What is the intention of this blank line break?

**Review by Itsuki54 — APPROVED** [review]
I'm LGTM, but I have a question.

File: packages/theme/src/components/menu.ts (hunk 1 above)
Comment (hirotomoyamada) [reply packages/theme/src/components/menu.ts]: Unnecessary! I'm surprised you noticed! Thanks!

**Review by Itsuki54 — APPROVED** [review]
thank you fix
LGTM

**Pull request merged**